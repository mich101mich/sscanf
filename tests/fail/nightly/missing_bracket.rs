fn main() {
    sscanf::sscanf!("hi", "asdf{asdf");
    sscanf::sscanf!("hi", "asdf}asdf");
}
