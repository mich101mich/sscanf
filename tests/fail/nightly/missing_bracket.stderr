error: missing '}' to close a placeholder. If the '{' was intended to be a literal, escape it with '{{'
 --> tests/fail/nightly/missing_bracket.rs:2:31
  |
2 |     sscanf::scanf!("hi", "asdf{asdf");
  |                               ^^^^^

error: Unexpected standalone '}'. Literal '}' need to be escaped as '}}'
 --> tests/fail/nightly/missing_bracket.rs:3:31
  |
3 |     sscanf::scanf!("hi", "asdf}asdf");
  |                               ^
